model/uzivatel.php

```diff
@@ -261,6 +261,49 @@ function jmeno()
     return $this->u['jmeno_uzivatele'].' '.$this->u['prijmeni_uzivatele'];
   }
 
+  /** Jméno uživatele */
+  function getJmeno()
+  {
+    return $this->u['jmeno_uzivatele'];
+  }
+  
+  /** Příjmení uživatele */
+  function getPrijmeni()
+  {
+    return $this->u['prijmeni_uzivatele'];
+  }
+  
+  /** Ulice a číslo popisné uživatele */
+  function getUlice()
```

Comment:
Podobně k principu výš: adresu bych měnil jako jednu věc ve view a tu pro ni udělal jen metodu `adresa`. Může to být klidně jen string, v případě dalšího použití se z toho pak může stát objekt s implementovanou metodou `__toString` a jednotlivými částmi adresy ve chvíli, kdy je budeme potřebovat jednotlivě.